src/Dfc.CourseDirectory.WebV2/Features/ProviderDashboard/Dashboard.cshtml

```diff
@@ -285,6 +285,28 @@ else if (Model.BulkUploadReadyToGoLiveApprenticeshipsCount > 0)
             </div>
         </div>
     </div>
+
+    <div class="govuk-grid-column-one-third">
```

Comment:
Need to wrap this in a check for the feature flag e.g.
```csharp
@if (FeatureFlagProvider.HaveFeature(FeatureFlags.DataManagement)
{
```

Reply:
Done.